### PR TITLE
Added account model

### PR DESCRIPTION
**Pull request by RSchipper**

### CODE REVIEW

**Comment by roelofr** [comment]
Duuuuude, die staat toch al in #6...

**Comment by RSchipper** [comment]
Player !== Account

File: database/seed/20170314120250-account.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ *
+ * @author Remco Schipper <github@remcoschipper.com>
+ */
+const bcrypt = require('bcrypt')
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    return queryInterface.bulkInsert('accounts', [{
+      name: 'Brian',
+      email: 'brian-the-second@king.com',
```
Comment (roelofr) [inline database/seed/20170314120250-account.js]: `@king.kong`?

File: .travis.yml [hunk 2]
```diff
@@ -15,3 +15,13 @@ notifications:
...
+
+env:
+  - CXX=g++-4.8
+  
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - g++-4.8
```
Comment (roelofr) [inline .travis.yml]: +2 min compiletime :sleeping:

**Pull request merged**